frontend/src/static/js/components/webstatus-overview-filters.ts

```diff
@@ -192,6 +230,141 @@ export class WebstatusOverviewFilters extends LitElement {
     window.location.href = newUrl;
   }
 
+  protected firstUpdated(): void {
+    if (this.apiClient !== undefined) {
+      // Perform any initializations once the apiClient is passed to us via context.
+      // TODO. allFeaturesFetcher should be moved to a separate task.
+      this.allFeaturesFetcher = () => {
+        return this.apiClient!.getAllFeatures(
+          getSearchQuery(this.location) as FeatureSearchType,
+          getSortSpec(this.location) as FeatureSortOrderType,
+          getWPTMetricView(this.location) as FeatureWPTMetricViewType
+        );
+      };
+    }
+  }
+
+  async exportToCSV(
+    completedCallback: (() => void) | undefined
```

Comment:
I don't think we need the `completedCallback` anymore, right?

Reply:
right.